### PR TITLE
Package breaches list into addon. r=groovecoder

### PR DESCRIPTION
**Pull request by nhnt11**

### CODE REVIEW

File: src/privileged/FirefoxMonitor.jsm [hunk 1]
```diff
@@ -136,8 +136,10 @@ this.FirefoxMonitor = {
       }
     }
 
-    XPCOMUtils.defineLazyPreferenceGetter(this, "breachListURL",
-      this.kBreachListURLPref, this.kDefaultBreachListURL);
```
Comment (groovecoder) [inline src/privileged/FirefoxMonitor.jsm]: Let's go ahead and remove the mention of haveibeenpwned.com completely.

Comment (nhnt11) [reply src/privileged/FirefoxMonitor.jsm]: done

**Comment by groovecoder** [comment]
I'm not able to spot-check this ... 

With `npm start` and debugging the add-on I see: `browser.fxmonitor is undefined background.js:5:1`

**Comment by groovecoder** [comment]
Oops, I was running with branded Firefox which didn't allow the un-signed addon. Changed to Nightly and it worked.

**Pull request merged**